client/packages/common/src/utils/functions/ApiUtils.tsx

```diff
@@ -2,9 +2,9 @@ export const setNullableInput = <NODE, KEY extends keyof NODE>(
   key: KEY,
   entity?: NODE | null | undefined
 ) => {
-  if (!entity) return undefined;
-
   if (entity == null) return { value: undefined };
 
+  if (!entity) return undefined;
```

Comment:
the previous order would not have allowed removal - if entity was `null` then it would have returned `undefined` which is 'unset' i.e. no change.